tests/behat/features/Express/cu_wysiwyg/cu_wysiwyg_pathologic.feature

```diff
@@ -1,10 +1,11 @@
 @wysiwyg @core
 Feature: WYSIWYG Pathologic tests
 
-@javascript @broken
-# The hardcoded url 'http://127.0.0.1:8031/testpage' will likely change in different setups.
-# This means the test is likely to fail if hosting details we do not control are different.
-# TO DO: Need to setup Pathologic link in testing module install.
+# Pathologic replaces the specified URL(s) with whatever the BaseURL is.
+# Since that changes with each install, we'll test that the specified URL is no longer there
+# TO Do: insert real baseURL.
+
+@javascript @mytest
```

Comment:
@cathysnider Did you put the '@myTest' tag here? Or did I at some point? It should probably be removed.